egs/babel/s5d/run-4-anydecode.sh

```diff
@@ -534,10 +552,14 @@ if [ -f exp/$chain_model/final.mdl ]; then
   decode=$dir/decode_${dataset_id}
   decode_script=steps/nnet3/decode.sh
 
-  if [ ! -f exp/nnet3$parent_dir_suffix/ivectors_${dataset_id}/.done ] ; then
-    steps/online/nnet2/extract_ivectors_online.sh --cmd "$decode_cmd" --nj $my_nj \
-      ${dataset_dir}_hires exp/nnet3$parent_dir_suffix/extractor exp/nnet3$parent_dir_suffix/ivectors_${dataset_id}/ || exit 1;
-    touch exp/nnet3$parent_dir_suffix/ivectors_${dataset_id}/.done
+  ivec_param=
+  if [[ $chain_model !=  *noivec* ]] ; then
```

Comment:
Is this and other similar diffs containing "noivec" still relevant?  Because I don't see where this suffix appears in any example script that is training a model.  Perhaps this relates to some of your experiments that were not checked in?